rsmtool/configuration_parser.py

```diff
@@ -928,61 +808,6 @@ def _parse_json_file(self, filepath):
 
         return configdict
 
-    def _parse_cfg_file(self, filepath):
```

Comment:
I still see a call to `_parse_cfg_file` in the if statement in `ConfigurationParser.parse()`. We should remove that.